packages/reports/src/Containers/open-positions.jsx

```diff
@@ -390,42 +396,56 @@ const OpenPositions = ({
         totals: active_positions_filtered_totals,
     };
 
+    const handleChange = e => {
+        setContractTypeValue(e.target.value);
+    };
     return (
         <React.Fragment>
             <NotificationMessages />
-            {has_multiplier_contract ? (
-                <Tabs
-                    active_index={active_index}
-                    className='open-positions'
-                    onTabItemClick={setActiveTabIndex}
-                    top
-                    header_fit_content={!isMobile()}
-                >
-                    <div label={localize('Options')}>
-                        <OpenPositionsTable
-                            className='open-positions'
-                            columns={columns}
-                            {...shared_props}
-                            row_size={isMobile() ? 5 : 63}
-                        />
-                    </div>
-                    <div label={localize('Multipliers')}>
-                        <OpenPositionsTable
-                            className='open-positions-multiplier open-positions'
-                            is_multiplier_tab
-                            columns={columns}
-                            row_size={isMobile() ? 3 : 68}
-                            {...shared_props}
+            {active_positions.length !== 0 && (
+                <>
```

Comment:
use `React.Fragment` to follow the standard in the code.

Reply:
Done!